src/components/bible/note-editor.tsx

```
@@ -0,0 +1,163 @@
+                            <MarkToolbarButton nodeType="italic" tooltip="Italic (⌘+I)">I</MarkToolbarButton>
+                            <MarkToolbarButton nodeType="underline" tooltip="Underline (⌘+U)">U</MarkToolbarButton>
+                        </FixedToolbar>
+                        <EditorContainer>
+                            <Editor placeholder="Type your amazing content here..." />
+                        </EditorContainer>
+                    </Plate>
+                )}
+            />
+            <form.Subscribe
+                selector={(state) => [state.canSubmit, state.isSubmitting]}
+                children={([canSubmit, isSubmitting]) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)